src/context-menu/context-menu.m.css

```diff
@@ -0,0 +1,13 @@
+.root {
+	position: absolute;
+	z-index: 11;
+}
+
+.underlay {
```

Comment:
is this still needed? I'd have imagined that the popup deals with the underlay?

Reply:
Removed